samples/openapi3/client/petstore/python-aiohttp/petstore_api/models/color.py

```diff
@@ -30,11 +32,11 @@ class Color(BaseModel):
     RGB array, RGBA array, or hex string.
     """
     # data type: List[int]
-    oneof_schema_1_validator: Optional[conlist(conint(strict=True, le=255, ge=0), max_items=3, min_items=3)] = Field(None, description="RGB three element array with values 0-255.")
+    oneof_schema_1_validator: Optional[Annotated[List[Annotated[int, Field(le=255, strict=True, ge=0)]], Field(min_items=3, max_items=3)]] = Field(default=None, description="RGB three element array with values 0-255.")
```

Comment:
I do not get an error if I construct this with `oneof_schema_1_validator=[1, 2]`. Maybe pydantic is thrown off by `Field()` being used on both sides of `=`? Also pydantic might not take into account `Field()` restrictions in the first argument of `Annotated[]`.

Best add a unit-test to make sure validation works, or is there one already?

Reply:
I will check this :+1:

Reply:
I added more tests: this works with Pydantic v2 but not with Pydantic v1. We'll focus on the former anyway and I'll merge my PR in a Pydantic v2-only generator :+1:

Reply:
This is great, thanks a lot for moving forward to pydantic v2! Then things work of course :)